src/custom/state/orders/updaters/UnfillableOrdersUpdater.ts

```diff
@@ -78,7 +79,15 @@ export function UnfillableOrdersUpdater(): null {
       const isUnfillable = isOrderUnfillable(order, price)
 
       // Only trigger state update if flag changed
-      order.isUnfillable !== isUnfillable && setIsOrderUnfillable({ chainId, id: order.id, isUnfillable })
+      if (order.isUnfillable !== isUnfillable) {
+        setIsOrderUnfillable({ chainId, id: order.id, isUnfillable })
```

Comment:
good call moving this inside the if, I hate statements that looks like an IF condition